accelerator/common_core.c

```diff
@@ -349,7 +349,10 @@ status_t ta_find_transaction_objects(const iota_client_service_t* const service,
                            NUM_TRITS_SERIALIZED_TRANSACTION, NUM_TRITS_SERIALIZED_TRANSACTION);
       ret = cache_set(txn_hash, cache_value);
       if (ret != SC_OK) {
-        goto done;
+        if (ret != SC_CACHE_OFF) {
+          goto done;
+        }
+        ret = SC_OK;
```

Comment:
I don't get it here. If it has error code, why the process keep going instead of `goto done`?

Reply:
Since the only error code allow to keep going here is `SC_CACHE_OFF` which is just a code to inform that the cache server is off. And in this situation, it is totally OK to do the following procedure as usual.

Reply:
I remembered I suggested this before, we could return `SC_OK` from cache if it's off instead of `SC_CACHE_OFF`. Then we could keep the branch condition simple here.

Reply:
yeah but it will turn that we have to add more detection somewhere else for recognizing if the cache server is turning on or not.
So if a return value of `SC_OK` is really necessary here, I would recommend to open a new PR for a refactoring which could solve this problem.